docs/orchestration/nextflow/nextflow-overview.md

```diff
@@ -278,7 +278,8 @@ The `process` definitions in Nextflow scripts should include a couple key parts
 * the `container` directive
 * `cpus` and `memory` directives to define resource that will be used by Batch Jobs
 
-An example defintion for a simple "Hello World" process is shown below:
+!!! note: The container image used to run a process needs to be capable of running the AWS CLI. It **doesn't** have to contain the CLI but it does need shared libraries such as `libz.so.1` which may not be present in very minimal images such as those based on Alpine.
```

Comment:
Needs to be compatible with `glibc`. Libraries like `libz.so.1` need not be present in the container - just readable from where ever they are mounted.

Reply:
Clarified in commit below